HSTracker/AppDelegate.swift

```diff
@@ -63,6 +63,24 @@ class AppDelegate: NSObject, NSApplicationDelegate {
                         }
                     }
                 }
+                
+                if oldSchemaVersion < 4 {
+                    // deck.version changes from string to two ints (major, minor)
+                    migration.enumerateObjects(ofType:
+                    Deck.className()) { oldObject, newObject in
+                        if let versionStr = oldObject?["version"] as? String {
+                            if let ver = Double(versionStr) {
+                                let majorVersion = Int(ver)
+                                let minorVersion = Int(ver - Double(majorVersion) * 10.0)
```

Comment:
I would change this line to `let minorVersion = Int((ver - Double(majorVersion)) * 10.0)` 
Because :
```
let ver: Double = 1.9
let majorVersion = Int(ver) // 1
let minorVersion = Int(ver - Double(majorVersion) * 10.0) // -8
```

```
let ver: Double = 1.9
let majorVersion = Int(ver) // 1
let minorVersion = Int((ver - Double(majorVersion)) * 10.0) // 9
```